include/seahorn/Houdini.hh

```diff
@@ -28,22 +28,32 @@ namespace seahorn
     virtual const char* getPassName () const {return "Houdini";}
   };
 
+  class GuessCandidates
```

Comment:
@Chenguang-Zhu , please re-factor `GuessCandidates` so that each guessing strategy is a separate class and that the choice of the strategy can be selected at runtime.